src/Cardano/CLI/Query.hs

```diff
@@ -157,21 +202,36 @@ queryVotingProportion
   :: ( MonadIO m
      , MonadReader backend m
      , BackendCompatible SqlBackend backend
-     , MonadError e m
-     , AsMetadataParsingError e
-     , AsMetadataRetrievalError e
      )
   => Api.NetworkId
   -> Maybe SlotNo
   -> Threshold
   -> m (Map (Api.StakeAddress, Api.Hash Api.StakeKey) Double)
 queryVotingProportion nw mSlotNo (Api.Lovelace threshold) = do
-  info <- filterAmounts (> threshold) <$> queryVoteRegistrationInfo nw mSlotNo
+  -- DECISION #08:
+  --   We successfully parsed a vote registration, but the voting
+  --   power associated with that registration does not exceed
+  --   the threshold.
+  --
+  --   Found entry
+  --   └── It had the right metadata keys
+  --       └── The metadata was JSON
+  --           └── The metadata was valid cardano-api TxMetadata
+  --               └── The metadata constituted a valid vote
+  --                   └── But the voting power did not exceed the threshold.
+  info <-  queryVoteRegistrationInfo nw mSlotNo
+
+  for_ (causeSumAmounts info) $ \(votepub, amt) -> do
+      if amt < fromIntegral threshold
+      then
+          liftIO $ hPutStr stderr $ notEnoughVotingPowerError votepub
```

Comment:
It's sort of odd that this is using hPutStr and the `\n` characters are in the message strings, but never mind.